cmd/daemon.go

```diff
@@ -88,7 +94,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	stats := metrics.NewStatistics()
+	stats, err := metrics.NewStatistics(pollPeriod)
```

Comment:
should the pollPeriod be `time.Duration` instead of `string` ?

Reply:
I wanted to handle the parsing of an integer and error handling within `NewStatistics`. 
I think its fine but willing to change it if you want.

Reply:
ur call in most of the application I saw this is normally used as duration so u can use `1s` or `1m` style I will leave it to you to decide